### PR TITLE
update project child process due the change of driver

### PR DESCRIPTION
**Pull request by jinghe-INTC**
verified with DCAP driver v1.33

### CODE REVIEW

**Comment by jinghe-INTC** [comment]
bors try

**Comment by jhand2** [comment]
Failure log here: https://oe-jenkins-dev.westeurope.cloudapp.azure.com/blue/organizations/jenkins/pipelines%2FAzure-Linux/detail/Azure-Linux/441/pipeline/

File: tests/child_process/host/host.cpp [hunk 1]
```diff
@@ -90,7 +92,8 @@ int main(int argc, const char* argv[])
             if (pid == 0) // child process
             {
                 result_child_process = oe_terminate_enclave(enclave);
-                OE_TEST(result_child_process == OE_INVALID_PARAMETER);
+                OE_TEST(result_child_process == OE_OK);
```
Comment (jhand2) [inline tests/child_process/host/host.cpp:95]: Given that the enclave is terminated in the parent and the child, I wonder if there is a race condition there. An OE enclave may not be able to be destroyed from both the parent and the child, I'm not sure.

That could be the cause of the build failure.

Comment (jinghe-INTC) [reply tests/child_process/host/host.cpp:95]: I verified the case on VM or FLC platform and didn't see any error with destroying.
And from the bors log, it shows the error is at Test failed at child process status check: 
> ../tests/child_process/host/host.cpp(113): main child_status == 0

So I guess it needs an "_exit(EXIT_SUCCESS);"  for the child process. I have added it and will start a bors try.

**Comment by jinghe-INTC** [comment]
bors try

File: tests/child_process/host/host.cpp (hunk 1 above)
Comment (jhand2) [reply tests/child_process/host/host.cpp:95]: oh, ya, that makes sense to me. Thanks!

**Pull request merged**